dashboard/app/models/circuit_playground_discount_application.rb

```diff
@@ -28,15 +28,14 @@ class CircuitPlaygroundDiscountApplication < ApplicationRecord
 
   enum unit_6_intention: {
```

Comment:
Note that I just looked at applications we've received over the past two years -- we've received a total of 6 of 700 that have answered something other than "Yes, I plan to teach in next two years".